### PR TITLE
Update installation instructions and main README

### PR DESCRIPTION
**Pull request by tonydewan**
The goal: be more accurate and informative, especially around installation. Also, fixes some formatting issues.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,90 +1,81 @@
-# Package Locations
+# Installation
 
-## PackageCloud
+Prebuilt `deb` and `rpm` packages are available via [packagecloud.io](https://packagecloud.io/). These files are also available to download directly, via the [releases page](https://github.com/Instrumental/instrumentald/releases).
```
Comment (janxious) [inline INSTALL.md]: The package cloud link should probably link to `https://packagecloud.io/app/expectedbehavior/instrumental/search?q=instrumentald&filter=`, or at least to the EB link.

File: INSTALL.md [hunk 2]
```diff
@@ -1,90 +1,81 @@
...
+```
 
- The Homebrew installer does not add `/etc/instrumentald.toml` or set `instrumentald` to run at startup. It's intended to make it easier to kick the tires, not for production metric gathering.
+The Homebrew formula does not add `/etc/instrumentald.toml` or set `instrumentald` to run at startup. It's intended to make it easier to kick the tires, not for production metric gathering.
 
- ## pkg
+### pkg
 
- Use pkg installer, then configure `/etc/instrumentald.toml`.
+Download the pkp from the [releases page](https://github.com/Instrumental/instrumentald/releases), run it, then edit `/etc/instrumentald.toml` to configure.
```
Comment (janxious) [inline INSTALL.md]: "pkp" -> "pkg"

File: INSTALL.md [hunk 3]
```diff
@@ -1,90 +1,81 @@
...
-* 64-bit RPM package (RHEL, Amazon AMI) [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_amd64.rpm](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_amd64.rpm)
-* 32-bit RPM package (RHEL, Amazon AMI) [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_i386.rpm](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_i386.rpm)
-* 64-bit Linux tarball (CoreOS, etc.) [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_linux-x86_64.tar.gz](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_linux-x86_64.tar.gz)
-* 32-bit Linux tarball (CoreOS, etc.) [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_linux-x86.tar.gz](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_linux-x86.tar.gz)
-* 64-bit Mac OS X tarball [https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_osx.tar.gz](https://s3.amazonaws.com/instrumentald/1.1.2/instrumentald_1.1.2_osx.tar.gz)
+Use pkg installer, then configure `/etc/instrumentald.toml` and run:
```
Comment (janxious) [inline INSTALL.md]: I think a header for this like _**Prebuilt `pkg` file**_ would be good, or given it's repeated below, delete?

File: INSTALL.md [hunk 4]
```diff
@@ -1,90 +1,81 @@
...
-
- ```
- brew install instrumental/instrumentald/instrumentald
- ```
+```
+brew install instrumental/instrumentald/instrumentald
+```
 
- The Homebrew installer does not add `/etc/instrumentald.toml` or set `instrumentald` to run at startup. It's intended to make it easier to kick the tires, not for production metric gathering.
+The Homebrew formula does not add `/etc/instrumentald.toml` or set `instrumentald` to run at startup. It's intended to make it easier to kick the tires, not for production metric gathering.
```
Comment (janxious) [inline INSTALL.md]: We should link to the homebrew formula/repo!

File: README.md [hunk 5]
```diff
@@ -46,7 +54,7 @@ ISD is built to make it easy to collect the most important metrics from your cri
 
 ## Custom Plugin Scripts
 
-Instrumental Daemon can monitor arbitrary processes and system events through a plugin scripting system. Writing plugins is easier than you'd think! Plugin script installation and development instructions are listed in [PLUGIN_SCRIPTS.md](PLUGIN_SCRIPTS.md).
+ISD can monitor arbitrary processes and system events through a plugin scripting system. Writing plugins is easier than you'd think! Plugin script installation and development instructions are listed in [PLUGIN_SCRIPTS.md](PLUGIN_SCRIPTS.md).
```
Comment (janxious) [inline README.md]: I could see making an explicit link to each of the samples here, since they are neat.

File: README.md [hunk 6]
```diff
@@ -79,6 +87,12 @@ The `start` command will start and detach the process. You may issue additional
 * `clean` - remove any files created by the daemon
 * `kill` - forcibly halt the daemon and remove its pid file
 
+By default, instrumentald will look for a configuration file at `/etc/instrumentald.toml`. You can pass an alternatve instead:
+
+```sh
+instrumentald -k <PROJECT_TOKEN> -H <HOSTNAME> -c <PATH_TO_CONF>
```
Comment (janxious) [inline README.md]: Are `-k` and `-c` mutually exclusive? If not, who wins in that scenario?

File: INSTALL.md (hunk 1 above)
Comment (tonydewan) [inline INSTALL.md]: Fixed

File: INSTALL.md (hunk 3 above)
Comment (tonydewan) [inline INSTALL.md]: updated to **`pkg` Installer**

File: INSTALL.md (hunk 4 above)
Comment (tonydewan) [inline INSTALL.md]: Done.

File: INSTALL.md (hunk 2 above)
Comment (tonydewan) [inline INSTALL.md]: Fixed.

File: README.md (hunk 5 above)
Comment (tonydewan) [inline README.md]: I added a clause to the sentence and link to that dir. Good enough?

**Comment by tonydewan** [comment]
@janxious I have addresses your concerns! 💃

File: INSTALL.md (hunk 3 above)
Comment (janxious) [inline INSTALL.md]: Right, but why is this here and also below?

Comment (tonydewan) [inline INSTALL.md]: WHOOPS. Fix'd.

**Comment by tonydewan** [comment]
@janxious I have actually now addressed all your concerns! 👯
